src/main/js/controls/Gantt/Gantt.js

```diff
@@ -204,14 +205,14 @@ class Gantt extends Construct {
                     ? this.config.canvasWidth
                     : this.config.canvasWidth - BASE_CANVAS_WIDTH_PADDING
             );
-        createDefs(this.config, this.svg);
         createGrid(this.axis, this.scale, this.config, this.svg);
         createContentContainer(this.config, this.svg);
         createAxes(this.axis, this.scale, this.config, this.svg);
         createGanttContent(this.config, this.svg);
         if (utils.notEmpty(this.config.dateline)) {
             createDateline(this.scale, this.config, this.svg);
         }
+        createDefs(this.config, this.svg);
```

Comment:
Based on your email: `I have changed the order in which the def is created because when the empty graph is loaded with dateline ( with panning enabled ) the createDefs is called before the createDateline so I changed the order to call the createDateline first then createDefs.`

What was the impact when panning is not used? 
What tests were breaking because of this?

Reply:
no test as such were breaking only those test cases where we are checking the order of the creation of the elements in the svg were breaking.